02-Git/RockPaperScissors.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27428.2037
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RockPaperScissors", "RockPaperScissors\RockPaperScissors.csproj", "{CA6DD72A-6D6A-4B86-8FCE-C6E2A624ADE7}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RockPaperScissorsTest", "RockPaperScissorsTest\RockPaperScissorsTest.csproj", "{54392AF4-6C91-468F-B7A1-474D22EA0734}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{CA6DD72A-6D6A-4B86-8FCE-C6E2A624ADE7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{CA6DD72A-6D6A-4B86-8FCE-C6E2A624ADE7}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{CA6DD72A-6D6A-4B86-8FCE-C6E2A624ADE7}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{CA6DD72A-6D6A-4B86-8FCE-C6E2A624ADE7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{54392AF4-6C91-468F-B7A1-474D22EA0734}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {10C356ED-1781-4B52-9CFE-CBBBABC7CB89}
+	EndGlobalSection
+EndGlobal
```

Comment:
Is this commit really needed?
If so, please include with the initial part of code